### ISSUE TITLE
csscalc always return false if usePrefixes is false

### ISSUE THREAD

**Issue by zenatti** [Modernizr/Modernizr#1852]
csscalc test always return false if I set usePrefixes false on modernizr 3.3.1

"el.style.cssText = prop + prefixes.join(value + prop);" doesn't check if Modernizr._config.usePrefixes is true or false.

**Comment by patrickkettner** [comment]
[I am not not able to replicate this](http://output.jsbin.com/ziboxulure/1)

can you give a bit more information?

**Comment by zenatti** [comment]
I have a build with usePrefixes:false

http://jsbin.com/leceyiyoti/edit?html,js,output

**Comment by patrickkettner** [comment]
ah, im very sorry. I messed up the config when I made the bin :(

the issue is that we are `join`ing the prefixes to get cssText, which is not working when prefixes is an empty array.

@ryanseddon @SlexAxton a hack fix would be to change prefixes from `[]` to `['','']`. Otherwise we will need to reformat a handful of detects to check that config value and react accordingly, or change them from something like

```js
var prop = 'width:';
var value = 'calc(10px);';
var el = createElement('a');

el.style.cssText = prop + prefixes.join(value + prop);
```

to something like

```js
var prop = 'width:';
var value = 'calc(10px);';
var el = createElement('a');

el.style.cssText = prop + value + prefixes.join(value + prop);
```

the `['','']`is the smallest amount of code, so I am leaning towards that. it just feels a bit dirty.

thoughts?

**Comment by SlexAxton** [comment]
I don't see a huge need to byteshave here. I'd be happy to to just check for the length of 'prefixes' and do something different in the zero case. But I'm not against `['','']` if you feel strongly.

**Comment by patrickkettner** [comment]
its a bit of a yak shaving moment, but the main reason I think about it is we would have to do the same check in every detect that uses `prefixes.join` method (about a dozen or so), or just short circuit the whole issue with a few chars.

if neither you or @ryanseddon are against the `['','']` way, i'll probably go that path with a comment

**Comment by ryanseddon** [comment]
Nope not against it.

**Comment by SlexAxton** [comment]
:+1:

**Comment by patrickkettner** [comment]
@zenatti fixed in master, should be released as 3.4.0 in a few days

**Comment by zenatti** [comment]
Thanks @patrickkettner !!
